Client/Frontend/Autofill/CreditCard/ViewComponents/CreditCardAutofillToggle.swift

```diff
@@ -15,16 +34,34 @@ struct CreditCardAutofillToggle: View {
                 .padding(.leading, 16)
                 .hidden()
             HStack {
-                Toggle(isOn: $toggleState) {
-                    Text("Save and autofill cards")
-                }.font(.system(size: 17))
-                 .padding(.leading, 16)
-                 .padding(.trailing, 16)
+                Toggle("Save and autofill cards", isOn: $model.isEnabled)
```

Comment:
nit: If it's not too much trouble, maybe we can change this string to `ToggleToAllowAutofillTitle`